ui/components/canvas2d/src/Canvas2D.svelte

```diff
@@ -1073,7 +1075,7 @@
 </script>
 
 <div
-  class={cn("flex h-full w-full bg-slate-100 transition-opacity duration-300 delay-100", {
+  class={cn("flex h-full w-full bg-gray-900 transition-opacity duration-300 delay-100", {
```

Comment:
`Canvas2D.svelte:1078`, `ImageCanvas.svelte:59`: `bg-slate-800` seems closer to the color used in the Sketch template than `bg-gray-900`, and would be more coherent with the rest of the app's color palette (92 uses of slate vs 7 uses of gray).

The remaining uses of gray could also be updated to use slate instead.